lib/urls.js

```diff
@@ -1,23 +1,24 @@
 'use strict';
-var parseQueryString = require('query-string').parse;
+
+const parseQueryString = require('query-string').parse;
+const isYoutubeRegex = /^(https?:\/\/)?(www\.)youtube\.com\//;
+const isAnImageRegex = /\.(gif|jpg|jpeg|tiff|png)$/i;
```

Comment:
This is the current behavior in all viewer variations. Checking the mime type is not appropriate for carousel it will take too long.